### DISCUSSION TITLE
Store result sparql construct

### DISCUSSION THREAD

**Discussion by RichDijk** [oxigraph/oxigraph#287] [Q&A] [answered]
Storing a graph into a store is going well, even very large files.
But how do I store a sparql construct result in a Store or Rdflib graph?

I have tried many options, one was:
```
for result in construct_result:
   store.add((Quad(NamedNode(result[0]), NamedNode(result[1]), NamedNode(result[2]))
```
But a type error occurs.

Thank you.

**Comment by Tpt** [comment, marked as answer]
@RichDijk If you cast all terms to `NamedNode`, it's not surprising a type error occur. This should work:
```python
for result in construct_result:
   store.add(Quad(result.subject, result.predicate, result.object))
```

**Reply by RichDijk** [reply]
That works nicely, thank you.
